### PR TITLE
polymer++

### PR DESCRIPTION
**Pull request by notwaldorf**
`x-repeat` -> `dom-repeat`
`x-autobind` -> `dom-bind`
`version++`
`polymer++`

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -2,12 +2,12 @@
...
-    "polymer": "polymer/polymer#v0.8.0-rc.7",
-    "iron-icon": "polymerelements/iron-icon#^0.8.0",
-    "iron-iconset-svg": "polymerelements/iron-iconset-svg#^0.8.0"
+    "polymer": "polymer/polymer#v0.9.0-rc.1",
+    "iron-icon": "polymerelements/iron-icon#^0.9.0",
+    "iron-iconset-svg": "polymerelements/iron-iconset-svg#^0.9.0"
   },
   "devDependencies": {
-    "paper-styles": "polymerelements/paper-styles#^0.8.0",
+    "paper-styles": "polymerelements/paper-styles#^0.9.0",
```
Comment (cdata) [inline bower.json]: Missing `iron-meta` devDependency.

Comment (cdata) [inline bower.json]: Also missing `"main"`

Comment (notwaldorf) [inline bower.json]: Done!

**Comment by cdata** [comment]
LGTM :+1:

**Pull request merged**